tools/schema_tools/gen_output.py

```diff
@@ -3,6 +3,7 @@
 import MySQLdb as mysqldb
 import os
 from socket import gethostname
+from lsst.sims.utils import photo_m5
```

Comment:
The `flat_sed` here refers to the sed of the astrophysical point source. Correct?